lib/dry/logic/rule.rb

```diff
@@ -48,8 +48,8 @@ def new(predicate)
         self.class.new(predicate, options)
       end
 
-      #bit of a hack, essentially genuine proc.curry should be provided it's args via[] whereas ours
-      #accepts them as a method param
+      # bit of a hack, essentially genuine proc.curry should be provided it's args via[] whereas ours
```

Comment:
This comment actually relates to the private `curried_predicate` method. Can you move it?